CanalTP/stat-compiler
# Remove journey_cities consolidation, since it is unused

How to test

* Launch a stat-compiler, and see in the logs that journey_cities is not consolidated anymore.

Note: I haven't added the migration to drop the tables so that I can backup the data before actually dropping it
